src/pl/sys_package/ob_dbms_workload_repository.cpp

```diff
@@ -1050,6 +1407,15 @@ int ObDbmsWorkloadRepository::print_ash_summary_info(const AshReportParams &ash_
     } else if (OB_FAIL(
                    temp_string.append_fmt("           Elapsed Time: %ld \n", dur_elapsed_time))) {
       LOG_WARN("failed to assign Elapsed Time string", K(ret));
+    // } else if (OB_FAIL(temp_string.append_fmt("         WR Begin Time: %.*s \n",
```

Comment:
Could you please add a output demo for a situation where user input time period across ash and wr. What would the ash and wr begin/end time be？

Reply:
User Input Begin Time: 2024-06-17 10:30:45
User Input End Time: 2024-06-17 18:45:38
Analysis Begin Time: 2024-06-17 17:40:56
Analysis End Time: 2024-06-17 17:56:29
Elapsed Time: 932
AWR Begin Time: 2024-06-17 10:32:40
AWR End Time: 2024-06-17 16:43:11